documentation/rfc/doc-routing-algorithm.md

```diff
@@ -0,0 +1,148 @@
+# Documentation of the routing algorithm
+
+## The implemented algorithm
+
+### Abstract
+
+a quick view
+
+### Example 1
+
+![transit_map](transit_map.png)
+
+Line 1 (every 20 minutes):
+
+| A  | E  |
+|----|----|
+|7:50|8:50|
+|8:10|9:10|
+|8:30|9:30|
+
+Line 2 (every 10 minutes):
+
+| A  | B  | C  | D  |
+|----|----|----|----|
+|8:00|8:30|8:33|8:36|
+|8:10|8:40|8:43|8:46|
+|8:20|8:50|8:53|8:56|
+
+Line 3 (every 5 minutes):
+
+| D  | C  | B  | E  | F  |
+|----|----|----|----|----|
+|8:35|8:38|8:41|9:00|9:02|
+|8:40|8:43|8:46|9:05|9:07|
+|8:45|8:48|8:51|9:10|9:12|
+
+Line 4 (every 5 minutes):
+
+| B  | C  | F  | E  |
+|----|----|----|----|
+|8:35|8:40|9:00|9:02|
+|8:40|8:45|9:05|9:07|
+|8:45|8:50|9:10|9:12|
+
+Line 5 (every 20 minutes):
+
+| E  | F  |  G  |
+|----|----|-----|
+|9:05|9:07|10:00|
+|9:25|9:27|10:20|
+|9:45|9:47|10:40|
+
+Connections: only intra stop point, 2 minutes except for F 3 minutes.
+
+### RAPTOR and the specificities
```

Comment:
It would be nice to have a small part on Pareto objectives and how they are themselves lexicographic objectives.
This would help understand the "multi-objective on arrival time" in second pass' description.

Reply:
That's not on raptor, that's on the global objectives. Our implementation of raptor only optimize arrival datetime and number of transfers